lib/src/resolvers/cluster.ts

```diff
@@ -68,10 +76,10 @@ export async function getTopClusters(
  */
 export async function getMintableClusters(
   _: unknown,
-  { address }: { address: string },
+  { address, codeHash }: { address: string; codeHash?: string },
```

Comment:
"codeHash" was not defined in the "Query.mintableClusters" type in the schema.
You should define a input filter type for the function and add it to the "Query.mintableClusters" type.